### PR TITLE
Next/prev cursors refactoring

### PR DESCRIPTION
**Pull request by gmist**
I think it also should be compatible with existing code (with gae-init v3.2.0). WDYT?

### CODE REVIEW

**Comment by lipis** [comment]
I like it :) if it works out of the box with the old one.. will be nice..

**Comment by lipis** [comment]
screen shot 2015-07-23 at 22 25 41

**Comment by gmist** [comment]
I'm just a little bit experimenting with the types and forgot to change them. Now it should work.

File: main/api/helpers.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def handle_error(e):
     }), e.code
 
 
-def make_response(data, marshal_table, next_cursor=None, prev_cursor=None):
+def make_response(data, marshal_table, next_cursor=None):
```
Comment (lipis) [inline main/api/helpers.py]: just `cursors` here is enough

Comment (gmist) [inline main/api/helpers.py]: I leave it for backwards compatibility, if someone is using named parameters
``` python
make_response(data, table, next_cursor=my_cool_cursor)
```

Comment (lipis) [inline main/api/helpers.py]: I really doubt that with only three params and one optional that would be the case :)

File: main/api/helpers.py [hunk 2]
```diff
@@ -44,11 +44,15 @@ def make_response(data, marshal_table, next_cursor=None, prev_cursor=None):
         'result': map(lambda l: restful.marshal(l, marshal_table), data),
       }
     if next_cursor:
-      response['next_cursor'] = next_cursor
-      response['next_url'] = util.generate_next_url(next_cursor)
-    if prev_cursor:
-      response['prev_cursor'] = prev_cursor
-      response['prev_url'] = util.generate_next_url(prev_cursor)
+      if isinstance(next_cursor, dict):
+        response['next_cursor'] = next_cursor['next_cursor']
```
Comment (lipis) [inline main/api/helpers.py]: then maybe `cursors['next']` and `cursors['prev']`

Comment (gmist) [inline main/api/helpers.py]: +1 is a good idea

**Comment by lipis** [comment]
Something is wrong.. with the previous link.. to reproduce locally:

- Create 5 users
- [localhost:8080/admin/user/?limit=2](http://localhost:8080/admin/user/?limit=2)
- Click next
- Click prev

Shows different results and sometimes (if you press around) no results.. 

(not sure if it's part of this PR or the previous one

**Comment by gmist** [comment]
Hm... at the same time this problem does not reproduced if you use ordering, right?
[http://localhost:8080/admin/user/?order=-modified&limit=2](http://localhost:8080/admin/user/?order=-modified&limit=2)

**Comment by lipis** [comment]
yes.. that's why I haven't noticed it when testing the other day.. but when tested on something else realised it :)

**Comment by lipis** [comment]
Update the online version.. https://gae-init.appspot.com/user/?limit=4

**Pull request merged**